test/bracketsTest.js

```diff
@@ -0,0 +1,130 @@
+process.env.NODE_ENV = 'test';
+
+var chai     = require('chai');
+var chaiHttp = require('chai-http');
+var mongoose = require('mongoose');
+var Project = require('../models/project');
+var server   = require('../app.js');
+var should   = chai.should();
```

Comment:
is the "should" var used anywhere?

Reply:
it was removed, thx.

Reply:
It is called for chai library to allow chainable assertions.
As I know, you don't remove it :(
http://cahijs.com/guide/styles/

Reply:
My conclusion,
- res.should is working without import of chai.should.
- and global.should was assigned by someting automatically.
So, for this test, chai.should does not seem to be necessary for res.should.

Reply:
No. We should import 'chai.should'.
I got errors when I tested each tests without import of chai.should.

Reply:
Hunseopui-Mac-mini:WATT EatACarrot$ ./node_modules/.bin/nyc --reporter=txt ./node_modules/.bin/mocha --timeout=1000 ./test/appserverTest.js 


Listening on port 3002
  test app.js
    test 404 error
      1) should return 404 error
    test port success : normal case
      2) should test normal port
    test port error : negative case
      3) should test negative port
    test port error : NaN case
      4) should test NaN port

Reply:
issue occurs only for mac ?
for me no issues and for travis no issues too.
however, to support mac, 'chai should' can be added again.

Reply:
Not only for mac.
You can test it.
When running npm test, first test sets the global.should.
It makes It seem to be no problem when we remove chai.should.
Try only one test without chai.should then you can get errors.
```
./node_modules/.bin/nyc --reporter=txt ./node_modules/.bin/mocha --timeout=1000 ./test/appserverTest.js
```

Reply:
something = other test case.
Ok, chai,should was added.